MainApp/.gitignore

```diff
@@ -8,4 +8,4 @@ tmp/
 !.gitkeep
 dist
 .idea/
-public/downloads/*
```

Comment:
looks like the gamedev branch is 2 commits behind the master one of which contains the 'public/downloads/*' added to gitignore.